code/client/makepkginfo

```diff
@@ -385,6 +385,15 @@ def check_mode(option, opt, value, parser):
                                                             (opt, mode))
     setattr(parser.values, option.dest, value)
 
+def has_valid_install_critieria(catinfo):
```

Comment:
Checking the `PayloadIdentifier` key is simply enough. Should it fail if that key is absent or None? Or just produce a warning?